filebeat/inputsource/udp/server.go

```diff
@@ -0,0 +1,79 @@
+package udp
+
+import (
+	"net"
+	"time"
+
+	"github.com/elastic/beats/libbeat/common/atomic"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+// Config options for the UDPServer
+type Config struct {
+	Host           string        `config:"host"`
+	MaxMessageSize int           `config:"max_message_size"`
```

Comment:
Is there any bounds on these values? If so you should add a `validate` tags like `validate:"min:0 max:65535"` or `validate:"required"` (double check the exact ucfg syntax)